apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -358,6 +358,11 @@ export const ProgressDot = Radium(React.createClass({
 }));
 
 export default connect(state => ({
+  // We now use ProgressBubbles everywhere in course overview scenarios. Because
+  // we plan to also get rid of this component in the near future, rather than
+  // clean it up more thoroughly (and risk regressions), I'm just making it
+  // explicit that courseOverviewPage is false here.
+  courseOverviewPage: false,
```

Comment:
Potentially contentious/a little bit weird. I initially started making a cleanup of ProgressDot that accounted for the fact that courseOverviewPage is always false (since we finished the last redesign). However, there was a pretty fair amount of change, and I decided I was better off avoiding regressions and leaving this component messy, since it will be deleted after we're done with this next redesign.